onyo/commands/new.py

```diff
@@ -19,70 +19,119 @@
         metavar='TEMPLATE',
         required=False,
         type=template,
-        help='Name of the template to seed the new asset(s)'),
+        help="""
+            Name of a template to populate the contents of new assets.
+
+            This cannot be used with the ``--clone`` flag nor the ``template``
+            Reserved Key.
+        """
+    ),
 
     'clone': dict(
         args=('-c', '--clone'),
         metavar='CLONE',
         required=False,
         type=path,
-        help='Path to an asset to clone from'),
+        help="""
+            Path of an asset to clone.
+
+            This cannot be used with the ``--template`` flag nor the
+            ``template`` Reserved Key.
+        """
+    ),
 
     'edit': dict(
         args=('-e', '--edit'),
         required=False,
         default=False,
         action='store_true',
-        help='Open new assets in editor before creation'),
+        help="""
+            Open new assets in an editor.
+        """
+    ),
 
     'keys': dict(
         args=('-k', '--keys'),
         required=False,
         action=StoreKeyValuePairs,
         metavar="KEYS",
         nargs='+',
-        help=(
-            'Key-value pairs to set in the new asset(s). Multiple pairs can be '
-            'specified (e.g. key=value key2=value2). All fields that are part of '
-            'asset filenames (defined in .onyo/config under `onyo.assets.filename`) '
-            'are required. If the value `faux` is assigned to the key `serial`, '
-            'a random, repository-unique string will be filled in instead.')),
+        help="""
+            Key-value pairs to populate content of new assets.
+
+            Each key can key defined either 1 or N times (where N is the number
```

Comment:
```suggestion
            Each key can be defined either 1 or N times (where N is the number
```

Reply:
Thanks for catching this! Fixed and pushed.